src/app/phase-team-response/issues-faulty/issues-faulty.component.html

```diff
@@ -1,15 +1,18 @@
-<div>
-  <mat-grid-list cols="3" rowHeight="80px">
-    <mat-grid-tile>
-      <div class="grid-flush-left"><h1 class="mat-headline" style="margin: 0px">Faulty Issues</h1></div>
-    </mat-grid-tile>
-
-    <mat-grid-tile>
-      <mat-form-field class="full-grid-width">
-        <input matInput (keyup)="applyFilter($event.target.value)" placeholder="Search" />
+<mat-expansion-panel expanded (opened)="panelOpenState = true" (closed)="panelOpenState = false">
+  <mat-expansion-panel-header [expandedHeight]="'70px'">
+    <mat-panel-title> Faulty Issues </mat-panel-title>
+    <mat-panel-description>
+      <mat-form-field class="full-grid-width" *ngIf="panelOpenState">
+        <input
+          matInput
+          (click)="$event.stopPropagation()"
```

Comment:
These `$event.stopPropagation()` are so that the expansion panel won't close when clicked or hwen the enter or space key are pressed